shared/package.json

```diff
@@ -7,7 +7,7 @@
     "_node": "cross-env-shell BABEL_ENV=yarn babel-node --extensions='.tsx' --extensions='.js' ",
     "help": "yarn run _helper help",
     "lint": "yarn run lint-specific .",
-    "lint-specific": "cd .. && shared/node_modules/.bin/eslint --quiet --ext js --ext ts --ext tsx --ext d.ts",
+    "lint-specific": "eslint --cache --quiet --ext js --ext ts --ext tsx --ext d.ts --ext mobile.tsx --ext desktop.tsx",
```

Comment:
Why not use a single `--ext` option like `--ext js,ts,tsx,d.ts,mobile.tsx,desktop.tsx`?

Reply:
didn't try it